src/App.jsx

```diff
@@ -8,6 +9,7 @@ function App() {
     <div>
       <Header />
```

Comment:
create a home page in the pages folder and move these components to the home page.

Reply:
Done